cmake/bob.cmake

```diff
@@ -156,8 +156,8 @@ endmacro(bob_public_dep)
 function(bob_export_target tgt_name)
   install(TARGETS ${tgt_name} EXPORT ${tgt_name}-target
       RUNTIME DESTINATION bin
```

Comment:
I think we should also set this to `CMAKE_INSTALL_BINDIR` to keep things consistent.

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html